### DISCUSSION TITLE
How to make a logout

### DISCUSSION THREAD

**Discussion by FirelCrafter** [rundfunk47/stinsen#85] [Q&A] [answered]
Hi, tried to make logout by changing root, but it's not working, how to do it better?)

**Comment by LePips** [comment, marked as answer]
You can use notifications to change the root, but you ultimately just need anything to change that your main coordinator listens to so that it can change its own `root`.

Here is a minimal example using notifications: https://pastebin.com/FWjR9bwu

**Reply by FirelCrafter** [reply]
Thanks! 🥇
